tutorials/advanced_features/Backends for Sequence Execution.ipynb

```diff
@@ -1,510 +1,610 @@
 {
```

Comment:
<p>We rather say <code>QutipBackendV2</code> and should not reference <code>QutipEmulator</code> anymore.</p><p>Also, see my comment below on the relevance of <code>job_params</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/952/discussion/#comment-2444241355'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>